File: src/providers/claude-code.ts
/**
 * Claude Code agent configuration
 */

import type { AgentConfig } from "../types.js";

/**
 * Default configuration for Claude Code agent via claude-code-acp
 */
export const claudeCodeConfig: AgentConfig = {
  command: "npx",
  args: ["@sudocode-ai/claude-code-acp"],
  env: {},
};

File: python/src/acp_factory/providers/claude_code.py
"""
Claude Code agent configuration
"""

from acp_factory.types import AgentConfig

# Default configuration for Claude Code agent via claude-code-acp
claude_code_config = AgentConfig(
    command="npx",
    args=["@agentclientprotocol/claude-agent-acp"],
    env={},
)
